pages/Nvidia/_index.md

```diff
@@ -55,21 +56,47 @@ env = __GLX_VENDOR_LIBRARY_NAME,nvidia
 env = WLR_NO_HARDWARE_CURSORS,1
 ```
 
-{{< hint >}}If you encounter crashes in Firefox, remove the line `env = GBM_BACKEND,nvidia-drm`.
+{{< hint >}}If hyprland crashes on start or you encounter crashes in Firefox, you may want to try removing the line `env = GBM_BACKEND,nvidia-drm`.
 {{< /hint >}}
 
 {{< hint >}}If you face problems with Discord windows not displaying or screen sharing not working in Zoom, remove or comment the line `env = __GLX_VENDOR_LIBRARY_NAME,nvidia`.
 {{< /hint >}}
 
+Run `lspci | grep VGA`.
+The returned line will start with some digits in the form `00:00.0`.
+Run `ls -l /dev/dri/by-path`.
+You should see at least one file with a filename like `pci-0000:00:00.0-card`.
+The one matching the earlier pci entry should be a symbolic link to `card0`, `card1` or `card2`.
+Set the environment variable `WLR_DRM_DEVICES` to that filepath.
+
+For example, if I run `lspci | grep VGA` and get
+
+```shell
+01:00.0 VGA compatible controller: NVIDIA Corporation GM107 [GeForce GTX 750 Ti] (rev a2)
+```
+
+and when I run `ls -l /dev/dri/by-path` I see
+
+```ls
+lrwxrwxrwx 1 root root  8 Nov  7 07:02 pci-0000:01:00.0-card -> ../card0
+lrwxrwxrwx 1 root root 13 Nov  7 07:02 pci-0000:01:00.0-render -> ../renderD128
+```
+
+I would add
+
+```sh
+env = WLR_DRM_DEVICES,/dev/dri/card0
```

Comment:
Doesn't the `/dev/dri/by-path/pci-...` path work here? I assume there's no issue if you only have one GPU, but for multiple GPUs, the kernel might change their order and break the config unexpectedly.

Reply:
I didn't know that was possible. I'll change it.

Reply:
40200258ea53de6a1056fd19bbf42d282544bc26
This look good?

Reply:
Yeah, looks good.